src/api/nutrition.ts

```diff
@@ -1,54 +1,82 @@
 import axios, { AxiosRequestConfig } from 'axios';
 import localForage from 'localforage';
+import moment from 'moment';
+import cloneDeep from 'lodash/cloneDeep';
+
 import { showLoader } from '@/shared/decorators';
 
 import { appKey, appID } from '../../credentials';
-import { IDailyIntake, ISearchItem, IHeaders } from '../models/nutrition';
+
+import { IDailyIntake, IHeaders, FoodInfo, Food, Meal } from '@/models/nutrition';
+import { FoodListItem } from '@/components/modules/nutrition/add-food/food-list-item.model';
 
 import { dailyIntake } from '@/mocks/daily-intake';
 
 enum API_ENDPOINTS {
```

Comment:
I would move this enum to separate file so all API endpoints are in one place.

Reply:
Done